packages/tasit-contract-based-account/src/GnosisSafe.test.js

```diff
@@ -62,11 +67,6 @@ describe("GnosisSafe", () => {
   });
 
   describe("test cases that need ETH deposit to the contract-based account", async () => {
-    beforeEach("faucet", async () => {
-      await etherFaucet(provider, minter, GNOSIS_SAFE_ADDRESS, ONE);
```

Comment:
So this wasn't necessary?

Reply:
Because thart expectation is being already made by a `beforeEach` hook before that:     // Expect an already funded Gnosis Safe wallet

Reply:
And the Gnosis Safe wallet is now funded by the script that runs after migrations and before all test suites.

Reply:
Ah, got it. Thanks!

Reply:
Resolving this